src/XML/src/FMI3/fmi3_xml_model_structure.c

```diff
@@ -103,11 +120,21 @@ void fmi3_xml_get_continuous_state_derivatives_dependencies(fmi3_xml_model_struc
     fmi3_xml_get_dependencies(ms->continuousStateDerivativeDeps, startIndex, dependency, factorKind);
 }
 
+void fmi3_xml_get_clocked_states_dependencies(fmi3_xml_model_structure_t* ms,
+                                           size_t** startIndex, size_t** dependency, char** factorKind) {
```

Comment:
```suggestion
void fmi3_xml_get_clocked_states_dependencies(fmi3_xml_model_structure_t* ms,
        size_t** startIndex, size_t** dependency, char** factorKind)
{
```
For line continuation on function decl, try to double indent and put { on new line. I know old code didn't do it, so please fix that as well